Add development environment setup in the CONTIBUTE markdown and move the readme info (#167)
GeyserMC/GeyserWiki: README.md

        - title: PageChild sub link
          url: "page/pagetitle/#headerLink"
```

> ### GeyserWiki  development setup locally

Reviewer: ```suggestion
### GeyserWiki development - Local deployment 
```
double space; mayyybe better title? not fully sure about it tbh

> * Download Ruby and Take the Ruby+Devkit from this page; https://rubyinstaller.org/downloads/

Reviewer: ```suggestion
* Download and install Ruby using the Ruby+Devkit from https://rubyinstaller.org/downloads/.
```

> * Once installed you will have to Run 'ridk install', this will open the RubyInstaller console.

Reviewer: ```suggestion
* Once installed, run 'ridk install' in console to open the RubyInstaller.
```

> Recommend choosing option 3; MSYS and MINGW after that ruby has been installed.

Reviewer: ```suggestion
For this usecase, it is recommended to choose the third option, "MSYS and MINGW". After that, the Ruby installation is completed.
```
i'd mention that it's best for this usecase

* Open the GeyserWiki folder in VScode, in terminal you will have to install the bundle first 'bundle install' command.
installing for the first time can take a while.

> * Once installed you can now start the server locally with command 'bundle exec jekyll serve'

Reviewer: ```suggestion
* Once installed, you can start the webserver locally with the 'bundle exec jekyll serve' command in the VSC terminal. 
```
